Gems/Atom/Feature/Common/Code/Source/ProfilingCaptureSystemComponent.cpp

```diff
@@ -590,6 +600,50 @@ namespace AZ
             return captureStarted;
         }
 
+        bool ProfilingCaptureSystemComponent::BeginContinuousCpuProfilingCapture()
+        {
+            if (m_cpuDataSerializationInProgress.load())
+            {
+                AZ_TracePrintf(
+                    "ProfilingSystemCaptureComponent",
+                    "Cannot begin a continuous capture - another serialization is currently in progress\n");
+                return false;
+            }
+            return AZ::RHI::CpuProfiler::Get()->BeginContinuousCapture();
+        }
+
+        bool ProfilingCaptureSystemComponent::EndContinuousCpuProfilingCapture(const AZStd::string& outputFilePath)
+        {
+            if (m_cpuDataSerializationInProgress.load())
+            {
+                AZ_TracePrintf(
+                    "ProfilingSystemCaptureComponent",
+                    "Cannot end a continuous capture - another serialization is currently in progress\n");
+                return false;
+            }
+
+            const bool captureEnded = AZ::RHI::CpuProfiler::Get()->EndContinuousCapture(m_lastContinuousCapture);
+            if (!captureEnded)
+            {
+                AZ_TracePrintf("ProfilingCaptureSystemComponent", "Could not end the continuous capture, is one in progress?\n");
+                return false;
+            }
+
+            // cpuProfilingData could be 1GB+ once saved, so use an IO thread to write it to disk.
+            m_cpuDataSerializationInProgress.store(true);
+            auto thread = AZStd::thread(
```

Comment:
Do we not have a thread pool for enqueuing async work of this nature as opposed to spinning up a dedicated thread, or is the idea to keep work off the async thread pool for some reason?

Reply:
No specific reason for using an owned thread here - moved to `AZ::Job` which should avoid some of the overhead of spawning a new thread!